### PR TITLE
Fixes on sidenav implementation

### PR DESCRIPTION
**Pull request by niglesiaszingoni**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 2.10.1 (2020-11-06)
+
+### Bugfix
+- By default any item should be selected [niglesias]
```
Comment (mpellerin42) [inline CHANGELOG.md:4]:
```suggestion
- By default no item should be selected [niglesias]
```

File: projects/demo/src/app/app.component.html [hunk 2]
```diff
@@ -8,15 +8,17 @@
     (close)="toggleSideNav(false)"
     >
     <pa-side-nav-header>
-        <img src="./assets/p-angular.svg">
+         <a href="#" traverseTo="/" tabindex="-1">
+            <img src="./assets/p-angular.svg" alt="Logo" tabindex="0">
```
Comment (mpellerin42) [inline projects/demo/src/app/app.component.html:12]: you can use PaFocusable instead of manually setting the tabIndexes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # 2.10.1 (2020-11-06)
 
 ### Bugfix
-- By default any item should be selected [niglesias]
+- By default no item should be selected [niglesias]
```
Comment (mpellerin42) [inline CHANGELOG.md:4]: Sorry I didn't see this on first review, but you should be explicit about what you fixed:
```
- **Sidenav demo**:
  - By default no item should be selected [niglesias]
  - Clicking on pastanaga icon in the sidenav should display home page again [niglesias]
  - on mobile/tablet, selecting a sidenav item should close the menu [niglesias]
```

Comment (niglesiaszingoni) [reply CHANGELOG.md:4]: oh true, i wasn't sure to use sidenav since the fixes are not for the sidenav component, thank you!

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**